torch/testing/__init__.py

```diff
@@ -102,6 +102,16 @@ def get_all_math_dtypes(device):
 
     return dtypes
 
+def get_all_math_dtypes_and_complex(device):
+    dtypes = [torch.uint8, torch.int8, torch.int16, torch.int32, torch.int64,
+              torch.float32, torch.float64, torch.complex64, torch.complex128]
```

Comment:
can we start with all math dtypes and append the complex ones? I'm concerned this could get out of sync if someone adds new dtypes and forgets to update both lists.